features/mgmt_api/mgmt_api_topics.feature

```diff
@@ -0,0 +1,160 @@
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#",
+            "subscribers": [
+                {"user_id": "#users._id#"}
+            ]
+        }
+        """
+        Then we get response code 201
+        When we get "/topics"
+        Then we get existing resource
+        """
+        {
+        "_items" :
+            [
+                {
+                    "label": "topic1",
+                    "company": "#companies._id#",
+                    "topic_type": "wire",
+                    "query": "topic1",
+                    "is_global": true,
+                    "user": "#users._id#",
+                    "subscribers": [
+                        {"user_id": "#users._id#", "notification_type": "real-time"}
+                    ]
+                }
+            ]
+        }
+        """
+
+    Scenario: Delete a topic
+        When we post to this "/topics"
+        """
+        [{
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#"
+        }]
+        """
+        When we delete latest
+        Then we get response code 204
+
+    Scenario: Update a topic
+        When we post to this "/topics"
+        """
+        [{
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#"
+        }]
+        """
+        When we patch latest
+        """
+        {"label": "topic2"}
+        """
+        Then we get updated response
+        """
+        {"label": "topic2"}
+        """
+
+    Scenario: Topic with folder
+        Given newsroom "topic_folders"
+        """
+        [
+            {   
+                "_id":"619277ef8bbbbfac6034aac1",
+                "name": "test",
+                "section": "wire"
+            }
+        ]
+        """
+        Given newsroom "topics"
+        """
+        [
+            {
+                "label": "topic1",
+                "company": "619277ef8bbbbfac6034aab4",
+                "topic_type": "wire",
+                "query": "topic1",
+                "is_global": true,
+                "user": "619277ef8bbbbfac6034aab7",
+                "folder": "619277ef8bbbbfac6034aac1"
+            }
+        ]
+        """
+        When we get "/topics"
+        Then we get list with 1 items
+
+    Scenario: Validate if navigation exists
+        When we post to this "/topics"
+        """
+        {
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#",
+            "navigation": ["619277ef8bbbbfac6034aab7"]
+        }
+        """
+        Then we get response code 500
```

Comment:
The original API returned a response code of 400. Validation may not be working as intended here if it's raising an internal error

Reply:
yep, but when validations fails, we get code `500` see
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/a07636b8-af13-4ccc-ac5d-dc51da5ab3be" />